backend/typescript/services/implementations/EntityService.ts

```diff
@@ -0,0 +1,51 @@
+import { errorMonitor } from "node:events";
+import { MEntity, Entity } from "../../models/entity.mgmodel";
+import { IEntityService, EntityRequestDTO } from "../interfaces/IEntityService";
+
+
+export class EntityService implements IEntityService {
+
+    async getEntity(id: string): Promise<Entity> {
+        let entity: Entity | null;
+        entity = await MEntity.findById(id);
+
+        if (!entity) {
+            throw new Error(`Entity id ${id} not found.`);
+        }
+        return entity;
+    }
+
+    async getEntities(): Promise<Entity[]> {
+        return await MEntity.find();
+    }
+
+    async createEntity(entity: Entity): Promise<Entity> {
```

Comment:
Sorry, I overlooked this the first time and also forgot to update in Notion but could we follow the pattern in #6 (in `UserService`)? Argument would be `EntityRequestDTO`, return type would be `Promise<EntityDTO>`.  (the 2 types are the same except `EntityRequestDTO` does **not** contain `id`). Similar changes in the update method as well.

I know it's a lot of repetitive boilerplate and it doesn't make too much of a difference right now, but we want to allow users to have the flexibility to choose which fields from the model to include/exclude in the object passed to/from the controller, and having separate DTOs will allow us to do that.